src/epcpm/parameterstobitfieldsc.py

```diff
@@ -0,0 +1,504 @@
+
+        h_lines = [
+            f"#ifndef {include_guard}",
+            f"#define {include_guard}",
+            f"",
+            f'#include "interface.h"',
+            f"",
+            f"",
+        ]
+
+        if not self.skip_output:
+            for parameter_uuid, modbus_nodes in parameter_uuid_to_modbus_node.items():
+                members_interface_c_lines = []
+                members_info_c_lines = []
+
+                name_uuid = epcpm.pm_helper.convert_uuid_to_variable_name(
+                    parameter_uuid
+                )
+
+                # Generate the SunSpec related .c/.h rows.
+                builder = builders.wrap(
+                    wrapped=modbus_nodes["sunspec"],
+                    parameter_uuid_finder=self.sunspec_model.node_from_uuid,
+                )
+                more_c_lines, more_h_lines = builder.gen()
+                c_lines.extend(more_c_lines)
+                h_lines.extend(more_h_lines)
+
+                common_c_lines = []
+                # Generate C lines for SunSpec node
+                if "sunspec" in modbus_nodes:
+                    builder = builders.wrap(
+                        wrapped=modbus_nodes["sunspec"],
+                        parameter_uuid_finder=self.sunspec_model.node_from_uuid,
+                    )
+                    common_c_lines.extend(builder.gen_common_interface())
+                    members_interface_c_lines.extend(
+                        builder.gen_bitfield_members_interface()
+                    )
+                    members_info_c_lines.extend(builder.gen_members_interface())
+                else:
+                    common_c_lines.extend(builder.gen_default_common_interface())
+
+                # Generate C lines for static modbus node
+                if "staticmodbus" in modbus_nodes:
+                    builder = builders.wrap(
+                        wrapped=modbus_nodes["staticmodbus"],
+                        parameter_uuid_finder=self.staticmodbus_model.node_from_uuid,
+                    )
+                    common_c_lines.extend(builder.gen_common_interface())
+                    members_interface_c_lines.extend(
+                        builder.gen_bitfield_members_interface()
+                    )
+                    members_info_c_lines.extend(builder.gen_members_interface())
+                else:
+                    common_c_lines.extend(builder.gen_default_common_interface())
+
+                c_lines.extend(members_interface_c_lines)
+
+                # Generate the combined interface item code
+                c_lines.extend(
+                    [
+                        f"InterfaceItem_Bitfield interfaceItem_{name_uuid} = {{",
+                        [
+                            f".common = {{",
+                            [ccl for ccl in common_c_lines],
+                            f"}},",
+                        ],
+                        [micl for micl in members_info_c_lines],
```

Comment:
rename micl to members\_info\_c\_line?

Reply:
See [this comment](https://github.com/epcpower/pm/pull/25#discussion_r749690952).